### ISSUE TITLE
Too low contrast ratio in sidebar menu

### ISSUE THREAD

**Issue by PrebenAas** [PostHog/posthog#19576]
## Too low contrast ratio in sidebar menu.

Following WCAG standards, text should have a contrast ratio of at least 7:1 ratio for AAA. Your text is 1.74:1, which is quite bad. Please consider fixing this. 

![image](https://github.com/PostHog/posthog/assets/20104610/5002ffae-4145-4c09-aa38-6e0463104166)

**Comment by pauldambra** [comment]
Our storybook accessibility checker picks this up too... 

<img width="1314" alt="Screenshot 2024-01-04 at 10 55 14" src="https://github.com/PostHog/posthog/assets/984817/cf2f185e-e68c-4d40-93f9-b5bac3293322">
